contents/docs/session-replay/how-to-control-which-sessions-you-record.mdx

```diff
@@ -24,12 +34,12 @@ posthog.init('<ph_project_api_key>', {
     disable_session_recording: true,
     // ... other options
 })
-posthog.onFeatureFlags(function () {
-    if (posthog.isFeatureEnabled('your-feature-flag')) {
-        posthog.startSessionRecording()
-    }
+yourCode.eventHandler(function () {
+    posthog.startSessionRecording()
 })
 ```
 
+NB using PostHog feature flags as above will buffer recordings until the flag is enabled. Using your own code will not buffer recordings before the call to `startSessionRecording`.
```

Comment:
NB? 

Also I don't really know what "buffer" means here.

Reply:
ah...

with the disable and then call `startSessionRecording` approach we don't collect any recording data until you say "startSessionRecording` so for e.g. if using someone else's flag solution you only get the recording from the point you call `startSessionRecording`

With a linked flag we store the recording data in memory until we have flag data. Once we know if the flag is on for the user we either drop the data we'd stored and you don't get a recording. Or you get the recording from the point posthog-js loaded. So, you might get slightly more recording data using e.g. our flags vs a competitor

Reply:
I'll nuke it... too many maybes and possiblys